src/pages/commonPage.ts

```diff
@@ -0,0 +1,642 @@
+// Import data
+import type {PageWaitForSelectorOptionsState, WaitForNavigationWaitUntil} from '@data/types/playwright';
+
+import type {
+  BrowserContext, ElementHandle, JSHandle, FileChooser, Frame, Page, Locator,
+} from 'playwright';
+
+/**
+ * Parent page, contains functions that can be used in every page (BO, FO ...)
+ * @class
+ */
+export default class CommonPage {
+  /**
+   * Get page title
+   * @param page {Page} Browser tab
+   * @returns {Promise<string>}
+   */
+  async getPageTitle(page: Page): Promise<string> {
+    return page.title();
+  }
```

Comment:
Should migrate to locators at the same time ?

Reply:
I will try in a second time when the POC will be validated with blockwishlist.